### DISCUSSION TITLE
🗺️ What is the future of Milkdown?

### DISCUSSION THREAD

**Discussion by sindras** [Milkdown/milkdown#1880] [General]
### Initial checklist

- [X] I agree to follow the [code of conduct](https://github.com/Saul-Mirone/milkdown/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched [issues](hhttps://github.com/Saul-Mirone/milkdown/issues) and [discussions](https://github.com/Saul-Mirone/milkdown/discussions) and couldn’t find anything (or linked relevant results below)

### Problem

Hi! First of all, thank you for creating this amazing editor! I have tried multiple similar solutions in the past but I love how Milkdown was able to provide great defaults using Crepe without requiring a lot of configurations.

That being said, I noticed that since September there is no new release. There is no movement on the Project board. All incoming bug reports went unanswered, and only PRs with small fixes got merged without any comment. 

I love Milkdown and I am interested to contribute to the editor, but without clear signs of long-term support from the community, this makes it hard for me to share or recommend Milkdown to my workplace or friends or use it in any of my projects. Is there something we can do about it?

### Solution

Some proposals:

1. Maintain a regular patch releases for small fixes that the community had contributed.
2. Find and assign maintainer(s).
3. Mark some small issues with "beginner-friendly" label, put some notes/clues, and let the community grab them if you are busy.
4. etc

### Alternatives

The alternative, if this repository is no longer updated and no maintainer stepping up, is to archive or put a "hiatus" notice or recommend a similar library that you like.

**Comment by quank123wip** [comment]
I don't want to see this project to die and recently I will start to submit PRs, trying to provide new features and revive old features.

**Comment by Saul-Mirone** [comment]
Hi, thanks for your support to milkdown. I don't think this project will die soon. I had some personal health issues in past few months but I'm trying to get back. But your proposals also make sense. I think we can have a `beginner-friendly` label and more docs help to let the community to know how to contribute to this project.
